### PR TITLE
back api for student reactions 

### PR DESCRIPTION
**Pull request by rozchlastywacz**
I added simple api that allows student to send reactions to lecturer.
How it works:
- student sends reaction payload to server
- server responds student if his reaction is sccepted (in order to reduce reaction-spamming)
- server adds that reaction with proper timestamp to student object which also emits an event
- lecture object reacts on that event and send reaction payload to lecturer 
- lecturer gets reaction payload with reaction and student id

That is all.
Waiting for reviews <3
original

### CODE REVIEW

**Review by ostatni5 — APPROVED** [review]
Good job.
I can't find any errors in code :c 
obraz

File: server/api/websocket-api.md [hunk 1]
```diff
@@ -69,8 +69,8 @@
     {
         "event": "subscribe_student",
         "data": {
-            "student_id": "20d7a5c9-f4f6-45ca-87a4-39ac97df3499",
-            "lecture_link": "1765661"
+            "student_id": "4fef1e0f-e77f-4302-aab6-d0cc67811368",
+            "lecture_link": "8341568"
```
Comment (Derstilon) [inline server/api/websocket-api.md:73]: What this data is used for?

File: server/api/websocket-api.md [hunk 2]
```diff
@@ -210,3 +210,51 @@
...
+    }
+    ```
+
+-   **Payload Content Emitted On Event:** `student added reaction`
+
+    ```json
+    {
+        "event": "send_student_reaction",
+        "data": {
+            "reaction": "POG",
```
Comment (Derstilon) [inline server/api/websocket-api.md:254]: image

**Review by Derstilon — APPROVED** [review]
POG reactions

**Review by Petros9 — COMMENTED** [review]
seems alright to me

**Pull request merged**